src/main/java/ch/njol/skript/expressions/ExprTarget.java

```diff
@@ -18,12 +18,23 @@
  */
 package ch.njol.skript.expressions;
 
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Mob;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.event.entity.EntityTargetEvent;
+import org.bukkit.util.RayTraceResult;
+import org.jetbrains.annotations.Nullable;
+
```

Comment:
Can you revert the import reordering change? This will make it harder to merge due to conflicting changes of the import  if another PR were to modify this class as well.

Reply:
Like in prior cases, I don't think policing imports is worthwhile